core/shared/src/main/scala/org/http4s/UrlForm.scala

```diff
@@ -100,14 +99,15 @@ object UrlForm {
   def fromChain(values: Chain[(String, String)]): UrlForm =
     apply(values.toList: _*)
 
-  implicit def entityEncoder(implicit charset: Charset = `UTF-8`): EntityEncoder.Pure[UrlForm] =
-    EntityEncoder.stringEncoder
+  def entityEncoder(charset: Charset = Charset.`UTF-8`): EntityEncoder.Pure[UrlForm] =
+    EntityEncoder
+      .stringEncoder(charset)
       .contramap[UrlForm](encodeString(charset))
       .withContentType(`Content-Type`(MediaType.application.`x-www-form-urlencoded`, charset))
 
   implicit def entityDecoder[F[_]](implicit
       F: Concurrent[F],
-      defaultCharset: Charset = `UTF-8`,
+      defaultCharset: Charset = Charset.`UTF-8`,
```

Comment:
This shouldn't be an implicit any more right?

Reply:
Yeah, 💯 . I've just tried to fix `EntityEncoder` in this PR.